CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # @dynatrace-oss/dynatrace-mcp-server
 
+## Unreleased changes
+
+- Added Copilot Natural Language 2 DQL Endpoint
```

Comment:
I am not 100% sure if we should keep the "2" as a stand-in for "to" here, but I also lack better ideas 😅
Maybe something along the lines of

```suggestion
- Add endpoint to translate between natural language and DQL
```

Capitalization seems inconsistent with other entries though.

Plus, are entries in present tense ("Add XYZ") or past tense ("Added XYZ")?